koji_containerbuild/plugins/cli_containerbuild.py

```diff
@@ -135,12 +141,22 @@ def parse_arguments(options, args, flatpak):
     if build_opts.signing_intent and build_opts.compose_ids:
         parser.error(_("--signing-intent cannot be used with --compose-id"))
 
+    if build_opts.operator_csv_modifications_url and not build_opts.isolated:
+        parser.error(_("Only --isolated builds support option --operator-csv-modifications-url"))
+
+    if (
+        build_opts.operator_csv_modifications_url and
+        '://' not in build_opts.operator_csv_modifications_url
```

Comment:
should we actually check this? Isn't it possible that users will omit that ://part but still provide a valid url?

Reply:
they should provide full URL

Reply:
there is already precedence for this in koji-c kode. For me valid URL has defined protocol